certified-connectors/RamQuest Actions - UAT/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1557 @@
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.AppliesToType"
+        },
+        "code": {
+          "description": "Gets or Sets Code",
+          "type": "string"
+        },
+        "description": {
+          "description": "Gets or Sets Description",
+          "type": "string"
+        },
+        "rtf": {
+          "description": "Gets or Sets Rtf",
+          "type": "string"
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderCommitmentSchedB": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "sequence": {
+          "format": "int32",
+          "description": "Gets or Sets Sequence",
+          "type": "integer"
+        },
+        "loanSequenceId": {
+          "format": "int32",
+          "description": "Gets or Sets LoanSequenceId",
+          "type": "integer"
+        },
+        "survey": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.SurveyScheduleBItem2Type"
+        },
+        "standbyYear": {
+          "format": "int32",
+          "description": "Gets or Sets StandbyYear",
+          "type": "integer"
+        },
+        "signatory": {
+          "description": "Gets or Sets Signatory",
+          "type": "string"
+        },
+        "state": {
+          "description": "Gets or Sets State",
+          "type": "string"
+        },
+        "county": {
+          "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.County"
+        },
+        "propertyType": {
+          "description": "Gets or Sets PropertyType",
+          "type": "string"
+        },
+        "insured": {
+          "description": "Gets or Sets Insured",
+          "type": "string"
+        },
+        "borrower": {
+          "description": "Gets or Sets Borrower",
+          "type": "string"
+        },
+        "titleVestedIn": {
+          "description": "Gets or Sets TitleVestedIn",
+          "type": "string"
+        },
+        "assignedLanguage": {
+          "description": "Gets or Sets AssignedLanguage",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.StandardLanguage"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderCommitmentSchedCD": {
+      "description": "",
+      "type": "object",
+      "properties": {
+        "sequence": {
+          "format": "int32",
+          "description": "Gets or Sets Sequence",
+          "type": "integer"
+        },
+        "loanSequenceId": {
+          "format": "int32",
+          "description": "Gets or Sets LoanSequenceId",
+          "type": "integer"
+        },
+        "assignedLanguage": {
+          "description": "Gets or Sets AssignedLanguage",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PubCS.TitleOrder.Api.Models.StandardLanguage"
+          }
+        }
+      }
+    },
+    "PubCS.TitleOrder.Api.Models.TitleOrderCommitment": {
+      "description": "",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.